scripts/pyladies-twitter-scraper.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+
+"""
+script to generate PyLadies twitter .csv with the following information:
+- twitter handle
+- description
+- expanded tweet profile url
+- date of last tweet
+
+To use you'll need to register a Twitter app at developer.twtter.com for the follwing values:
+- CLIENT_ID
+- CLIENT_SECRET
+- ACCESS_TOKEN
+- ACCESS_TOKEN_SECRET
+"""
```

Comment:
Adding a brief usage command about how to run will help newcomers to the script. `python pyladies-twitter-scraper.py`